src/Dockerfile

```diff
@@ -14,10 +14,12 @@ RUN apt-get update -qy && apt-get install -y wget
 RUN apt-get update -qy
 RUN apt-get -qq -y update && \
     DEBIAN_FRONTEND=noninteractive apt-get -qq -y install \
-    python3-dev python3-pip postgresql postgresql-contrib libffi-dev libpq-dev gcc musl-dev wkhtmltopdf sudo \
+    python3-dev python3-pip postgresql postgresql-contrib libffi-dev libpq-dev gcc musl-dev sudo \
     fonts-indic
 
 # install wkhtmltopdf
+RUN wget https://github.com/wkhtmltopdf/packaging/releases/download/0.12.6-1/wkhtmltox_0.12.6-1.focal_amd64.deb
+RUN apt-get -qq -y install ./wkhtmltox_0.12.6-1.focal_amd64.deb
```

Comment:
Why are we adding a deb to a container image?

Reply:
because we want latest version of wkhtmltopdf library and it is only available via deb from the official source